docs/UserGuide.md

```diff
@@ -735,6 +787,10 @@ Action | Format, Examples
 **Edit** | `editmodule n/MODULE_NAME e/NEW_NAME l/NEW_LINK`<br> e.g. `editmodule n/CS2103T e/CS2100 l/https://sample.zoom.us`
 **Add Zoom to module** | `addzoom INDEX n/LESSON_NAME z/ZOOM_LINK` <br> e.g. `addzoom 1 n/lecture z/https://nus-sg.zoom.us/j/uascya367yfy`
 **Add Assignment** | `addassignment n/MODULE_NAME a/ASSIGNMENT_NAME %/ASSIGNMENT_PERCENTAGE r/ASSIGNMENT_RESULT` <br> e.g. `addassignment n/CS2100 a/Quiz 1 %/5 r/0.80`
+**Archive** | `archivemodule MODULE_POSITION `<br> e.g. `archivemodule 3`
+**Un-archive** | `unarchivemodule MODULE_POSITION `<br> e.g. `unarchivemodule 3`
```

Comment:
I think we should standardise this, either use INDEX or MODULE_POSITION across the board.

Reply:
Alright thanks for the reminder.